src/arr/desugar/ds-main.arr

```diff
@@ -0,0 +1,30 @@
+provide {
+    desugar: desugar,
+    resugar: resugar
+} end
+
+import ast as AST
+import ast-visitors as AV
+import file as F
+import file("conversion-visitor.arr") as CONV
+import file("ds-structs.arr") as S
+import file("ds-sugar.arr") as DS
+import file("ds-parse.arr") as P
+
+desugaring-rules = block:
+  file = F.input-file("src/arr/desugar/pyret.sugar")
```

Comment:
@jpolitz right now we are reading the sugars like this. Will this be a problem on CPO (which doesn't have filesystem)?